CHANGELOG.md

```diff
@@ -17,6 +17,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+## [2.3.0]
+### Added
+ - Added the following colors: `pink100`, `pink200`, `pink300`, `pink500`, `pink600`, `orange100`, `orange200`, `orange300`, `orange500`, `orange600`, `royal100`, `royal200`, `royal300`, `royal500`, `royal600`, `royal700`
+
+### Changed
+  - Changed the value of the following colors: `yellow500`, `pink400`, `orange400`, `royal400`
+
+### Removed
```

Comment:
This feels like a breaking change to me so we might want to call this v3

Reply:
Ah I see you did move it up to v2, but without starting over at `2.0.0` which we will want to do

Reply:
I'm not sure if we up the version in `package.json` in this PR or after it goes to `master` and if we use the `npm version` command for that or if that has any unwanted side effects. This repo could really use some contributing instructions. Maybe I'll add them after I publish this.

Reply:
I'm about to run into the same issue in `niche-logger` so I feel your pain